lstchain/scripts/lstchain_data_create_drs4_pedestal_file.py

```diff
@@ -100,13 +94,12 @@ def main():
     # Finalize pedestal and write to fits file
     pedestal.finalize_pedestal()
 
-    primaryhdu = fits.PrimaryHDU(event.lst.tel[tel_id].svc.pixel_ids)
-    secondhdu = fits.ImageHDU(np.int16(pedestal.meanped))
-
-    hdulist = fits.HDUList([primaryhdu, secondhdu])
+    expected_pixel_id = fits.PrimaryHDU(event.lst.tel[tel_id].svc.pixel_ids)
+    pedestal_mean_array = fits.ImageHDU(np.int16(pedestal.meanped))
+    failing_pixels = fits.ImageHDU(pedestal.failing_pixels_list)
```

Comment:
Why do you store the failing pixels as image HDU? Is that really the best representation?

Reply:
I changed to `BinTableHDU`